authors/apps/authentication/serializers.py

```diff
@@ -26,7 +28,7 @@ class Meta:
         # List all of the fields that could possibly be included in a request
         # or response, including fields specified explicitly above.
         # return a success message on succeesful registration
-        fields = ['email', 'username', 'password']
+        fields = ['email', 'username', 'password', 'token']
```

Comment:
@ssewilliam whay have u readded the token field here at the registration serializer?

Reply:
@ssewilliam  Reply to this